include/deal.II/lac/scalapack.h

```diff
@@ -189,6 +189,163 @@ class ScaLAPACKMatrix : protected TransposeTable<NumberType>
                const std::pair<unsigned int,unsigned int> &offset_B,
                const std::pair<unsigned int,unsigned int> &submatrix_size) const;
 
+  /**
+   * Transposing assignment: <i>A = B<sup>T</sup></i>
+   *
+   * The matrices <tt>A</tt> and <tt>B</tt> must have the same process grid.
+   *
+   * Following alignment conditions have to be fulfilled: $MB_A$ = $NB_B$ and $NB_A$ = $MB_B$
+   */
+  void copy_transposed(const ScaLAPACKMatrix<NumberType> &B);
+
+  /**
+   * Matrix-addition:
+   *
+   * <i>A = a A + b op(B)</i>
+   *
+   * if(transpose_B)  <i>op(B) = B<sup>T</sup></i>
+   *
+   * else  <i>op(B) = B</i>
```

Comment:
I'd prefer for this to be expressed in words rather than as pseudo-code. i.e.

```
If the @p transpose_B is set then the transpose of @p B is used.
That is to say that the operation $op(B)$ corresponds to $B^{T}$. 
If this flag is set to false, then $op(B)$ applies @p B with its default ordering.
```

This also applies below.

Reply:
I would favor the pseudo-code as  it is short and precise.
But I can adapt it if wanted.

Reply:
I suppose that the pseudo-code is consistent with [`LapackFullMatrix`](https://www.dealii.org/developer/doxygen/deal.II/classLAPACKFullMatrix.html#a2fb96068b464f9eaddccd74696bdaf42) and elements of the [`Vector` class](https://www.dealii.org/developer/doxygen/deal.II/classLinearAlgebra_1_1Vector.html#a5c4190172e7ebc876dbdcde8617813f3). Although I'm not a huge fan of it, I'm not going to push the issue since there's precedent for it.